src/collections/_documentation/platforms/go/panics.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Handling Panics
+sidebar_order: 3
+---
+
+The way to capture unhandled panics in our Go SDK is through `Recover` method. It can be either used directly through `defer` keyword, or as the part of your own implementation.
+
+## Usage
+
+When used directly as shown below, Sentry will recover from the panic and internally decide whether to use `CaptureException` or `CaptureMessage` method, based on the type of input it received.
+As it's not uncommon to panic with a string, it's recommended to use `AttachStacktrace` option during SDK initialization, which will try to provide an useful stack trace for messages as well.
+
+```go
+func() {
+	defer sentry.Recover()
+	// do all of the scary things here
+}()
+```
+
+By default, Sentry Go SDK uses asynchronous transport, which in the code example below requires an explicit awaiting for event delivery to be finished using `sentry.Flush` method. It is necessary, because otherwise the program would not wait for the async HTTP calls to return a response, and exit the process immediately when it reached the end of the `main` function. It would not be required inside a running goroutine or if you would use `HTTPSyncTransport`, which you can read about in `Transports` section.
+
+If you want to control the delivery for a single `defer` call, or do some other things before capturing, you have to use `Recovery` method on the `Hub` instance directly, as it can accept `err` itself.
```

Comment:
```suggestion
If you want to control the delivery for a single `defer` call, or do some other things before capturing, you have to use the `Recovery` method on the `Hub` instance directly, as it can accept `err` itself.
```